.github/workflows/build.yml

```diff
@@ -47,14 +47,14 @@ jobs:
             matrix: macos
             emoji: 🍎
             runs-on:
-              arm: [macOS, ARM64]
-              intel: [macos-12]
+              arm: [macos-13-arm64]
```

Comment:
This is switching to the GitHub hosted runner?  Is that the intent?

Reply:
I've been using macos-13-arm64 everywhere - it's used in chia blockchain for example

Reply:
Sounds like we should use what you changed it to here.  The existing pattern is still appropriate when we arent looking to call out a specific version.